src/components/workMenuEntry.svelte

```diff
@@ -70,6 +71,11 @@
     const target: HTMLInputElement = e.target as HTMLInputElement;
     const currValue = target ? target.value : '';
     if (e.key == 'Enter' && currValue.length > 0) {
+      // if the tag is not in the tags, add the tag into the store
+      if (getTag(currValue) == undefined) {
+        addTag(new Tag(currValue));
```

Comment:
I do not save the tags after adding the tag in this menu.

Which also makes me think... I don't like having to call `saveTags` each time I do some sort of operation on the tags. I should just have each of the manipulation functions call `saveTags` themselves so I don't have to worry about it anymore.